src/api/app/policies/label_template_policy.rb

```diff
@@ -0,0 +1,22 @@
+class LabelTemplatePolicy < ApplicationPolicy
+  class Scope
+    def initialize(user, project)
+      @user = user
+      @project = project
+    end
+
+    def resolve
+      return [] unless Flipper.enabled?(:labels, @user)
```

Comment:
I don't really see much of a point for this, you already have `authorize` in the view, why repeat it here?

Reply:
Right, the scope is not really needed